docs/docs/authentication/sign-in-methods/3-personal-access-tokens.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Sign In with Personal Access Tokens
+sidebar_label: Personal Access Tokens
+slug: /authentication/sign-in-with-personal-access-tokens
+---
+
+Nhost allows you to sign in users with personal access tokens (PAT) which is a way to sign in users without an email address or password.
+
+## Configuration
+
+:::info
+Personal Access Tokens can only be created through Hasura Auth or the [Nhost JavaScript SDK](/reference/javascript) at the moment. We have plans to add support for personal access tokens in the Nhost Dashboard in the future.
```

Comment:
I'd remove the second comment starting with "We have plans...", I am a bit unsure about this.